lambda/utils.go

```diff
@@ -35,6 +28,39 @@ type ECRAuth struct {
 	ExpiresAt     time.Time
 }
 
+// User parametes. Note that Keys should match to event parameters unless specifying separate json keys
+type UserParameters struct {
+	SrcImage  string
+	SrcCreds  string
+	DestImage string
+	DestCreds string
+}
+
```

Comment:
These seems like a refactor

Reply:
True, modified the code different way that I would not like, but increasing technical debt then :)